src/main/java/reposense/model/RepoConfiguration.java

```diff
@@ -123,19 +123,20 @@ public static void merge(List<RepoConfiguration> repoConfigs, List<AuthorConfigu
     public static void setGroupConfigsToRepos(List<RepoConfiguration> repoConfigs,
             List<GroupConfiguration> groupConfigs) {
         for (GroupConfiguration groupConfig : groupConfigs) {
+            List<RepoConfiguration> matchingRepoConfigs;
             if (groupConfig.getLocation().isEmpty()) {
-                continue;
+                matchingRepoConfigs = repoConfigs;
```

Comment:
Seems that it can be feature simplified with `? :`, but it is OK to leave it here.